contracts/implementations/MultipleMinter.sol

```diff
@@ -155,6 +155,7 @@ contract MultipleMinter is
      *  manufacturerNode -> Parent manufacturer node id of the vehicle
      *  owner -> The new nodes owner
      *  deviceDefinitionId -> The Device Definition Id
+     *  attrInfoPairsVehicle -> List of attribute-info pairs to be added of the vehicle
```

Comment:
Should we also/instead put these docs on the struct?

Reply:
Good stuff